mpdm/Propagation[] epochs
std_msgs/Float32 learning_time